pkg/helm3/build.go

```diff
@@ -94,15 +97,20 @@ func (m *Mixin) Build() error {
 		for _, name := range names {
 			url := input.Config.Repositories[name].URL
 			repositoryCommand, err := getRepositoryCommand(name, url)
-			if err != nil && m.Debug {
-				fmt.Fprintf(m.Err, "DEBUG: addition of repository failed: %s\n", err.Error())
+			if err != nil {
+				if m.Debug {
+					fmt.Fprintf(m.Err, "DEBUG: addition of repository failed: %s\n", err.Error())
```

Comment:
I'm not sure why this is hidden behind a debug flag? Seems like this will silently fail otherwise when they were relying on it to use the bundle. Should we return an error instead?